flocker/node/agents/ebs.py

```diff
@@ -15,9 +15,9 @@
 import boto3
 
 from botocore.exceptions import ClientError, EndpointConnectionError
-from botocore.vendored.requests.packages.urllib3.contrib.pyopenssl import (
-    extract_from_urllib3,
-)
+# from botocore.vendored.requests.packages.urllib3.contrib.pyopenssl import (
+#     extract_from_urllib3,
+# )
```

Comment:
Fine either deleting this or leaving the call in, but probably don't just comment it out.

Given the insecure SSL... maybe leave it in?